model-optimizer/extensions/middle/RandomUniformReplacer.py

```diff
@@ -18,13 +18,14 @@
 from typing import Dict
 
 from mo.front.tf.graph_utils import create_op_with_const_inputs
-from mo.graph.graph import Graph, Node
+from mo.graph.graph import Graph, Node, rename_nodes
 from mo.middle.replacement import MiddleReplacementPattern
 from mo.ops.broadcast import Broadcast
 
 
 class RandomUniformReplacer(MiddleReplacementPattern):
```

Comment:
Probably, we should remove this transformation. It was based on the same example (JIRA: 36877).

Reply:
Yes, you are correct. But I didn't remove it because I am not sure if it is only model using this transformation and the pattern is also the same.

Reply:
The old transformation was removed.